azurerm/internal/services/hdinsight/hdinsight_ml_services_cluster_resource.go

```diff
@@ -53,10 +53,11 @@ var hdInsightMLServicesClusterEdgeNodeDefinition = azure.HDInsightNodeDefinition
 
 func resourceArmHDInsightMLServicesCluster() *schema.Resource {
 	return &schema.Resource{
-		Create: resourceArmHDInsightMLServicesClusterCreate,
-		Read:   resourceArmHDInsightMLServicesClusterRead,
-		Update: hdinsightClusterUpdate("MLServices", resourceArmHDInsightMLServicesClusterRead),
-		Delete: hdinsightClusterDelete("MLServices"),
+		DeprecationMessage: "Due to HDInsight 3.6 retirement on December 31, 2020",
```

Comment:
can we make this:

```suggestion
		DeprecationMessage: `HDInsight 3.6 will be retired on 2020-12-31 - MLServices is not supported in HDInsight 4.0 and so this resource will be removed in the next major version of the AzureRM Terraform Provider.
		
More information on the HDInsight 3.6 deprecation can be found at:

https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-component-versioning#available-versions`,
```

(whilst the new-lines are probably overkill, it helps on smaller terminals)